standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/repl/ReplScope.java

```diff
@@ -116,4 +117,27 @@ private boolean inTableExcludedList(final String tableName) {
     }
     return excludedTableNamePattern.matcher(tableName).matches();
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (o == this) {
+      return true;
+    }
+    if (! (o instanceof ReplScope)) {
+      return false;
+    }
+
+    ReplScope replScope = (ReplScope)o;
+    return ((replScope.getExcludedTableNames() == null ?  (this.excludedTableNames == null) :
+      replScope.getExcludedTableNames().equals(this.excludedTableNames))
```

Comment:
nit: getExcludedTableNames() /getIncludedTableNames() methods can be replaced by replScope.excludedTableNamePattern and replScope.includedTableNamePattern respectively.